alws/models.py

```diff
@@ -1289,6 +1289,12 @@ class SignKey(PermissionsMixin, Base):
     inserted: Mapped[datetime.datetime] = mapped_column(
         sqlalchemy.DateTime, default=datetime.datetime.utcnow()
     )
+    active: Mapped[bool] = mapped_column(
+        sqlalchemy.Boolean, default=True
```

Comment:
Are you sure that we really need to set it to true by default?

Reply:
I believe so since a new key is always active.